gateway/conf.d/backend.conf

```diff
@@ -1,4 +1,11 @@
+{% if jaeger_enabled %}
```

Comment:
No liquid in this template (it would need to have `.liquid` extension). This will neeed to be in the main nginx.conf.liquid.